styles/styles.css

```diff
@@ -0,0 +1,311 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap");
+
+:root {
```

Comment:
We can go through and reduce these variables/the css later this week when the design is more fleshed out.